### ISSUE TITLE
Setup maven central integration to push client lib binaries to

### ISSUE THREAD

**Issue by anuragkapur** [interfax/interfax-java#1]

**Comment by anuragkapur** [comment]
Ticket raised with Sonatype: https://issues.sonatype.org/browse/OSSRH-25418

**Comment by anuragkapur** [comment]
done

**Comment by cbetta** [comment]
@anuragkapur can you please also ensure that someone from InterFAX has access to push? Probably @eyalnevo

**Comment by anuragkapur** [comment]
@cbetta - all commits to master are automatically built, testes, tagged and pushed to maven central if successful. Is that not sufficient?

So in summary, all with write access to master can push..

**Comment by cbetta** [comment]
@anuragkapur sure but using your account, right? What if they ever change brand, want to pull a release, etc?

**Comment by anuragkapur** [comment]
Currently, my account, yes. But they can be added whenever they like, given they can prove ownership of the domain.

I'll get them setup now, if you like. 

@if-adam , @eyalnevo - please can you signup for an account here: https://issues.sonatype.org/secure/Signup!default.jspa

Once done, I'll raise a ticket with the maven central team to ensure you have access to the maven project for interfax-java.

**Comment by cbetta** [comment]
Thank you @anuragkapur

**Comment by eyalnevo** [comment]
@anuragkapur - I have registered with Sonatype - our username is "InterFAX". Do you require the password as well?

**Comment by anuragkapur** [comment]
@eyalnevo - thanks. No, don't need the password. I've raised a request: https://issues.sonatype.org/browse/OSSRH-25640 to ensure you have full access to the maven project.

All that is required to push new artifacts to maven central are: 
- access to the maven project, which I have now requested for you
- gpg key setup as described here: http://central.sonatype.org/pages/working-with-pgp-signatures.html

The entire process is described here: http://central.sonatype.org/pages/ossrh-guide.html

Also, the entire end to end process is fully automated and in-place. As suggested by @cbetta , I am ensuring you have the necessary access, just in-case.
